products/compute/api.yaml

```diff
@@ -11316,6 +11317,60 @@ objects:
           PARTNER InterconnectAttachment is created, updated,
           or deleted.
         output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: bfd
+        description: |
+          BFD configuration for the BGP peering.
+        min_version: beta
+        properties:
+          - !ruby/object:Api::Type::Integer
+            name: minReceiveInterval
+            description: |
+              The minimum interval, in milliseconds, between
+              BFD control packets received from the peer router. The actual value
+              is negotiated between the two routers and is equal to the greater of
+              this value and the transmit interval of the other router. If set,
+              this value must be between 100 and 30000. The default is 300.
+            default_value: 300
+          - !ruby/object:Api::Type::Integer
+            name: minTransmitInterval
+            description: |
+              The minimum interval, in milliseconds, between
+              BFD control packets transmitted to the peer router. The actual value
+              is negotiated between the two routers and is equal to the greater of
+              this value and the corresponding receive interval of the other
+              router. If set, this value must be between 100 and 30000. The default
+              is 300.
+            default_value: 300
+          - !ruby/object:Api::Type::Integer
+            name: multiplier
+            description: |
+              The number of consecutive BFD packets that must be missed
+              before BFD declares that a peer is unavailable. If set, the value
+              must be a value between 2 and 16. The default is 3.
+            default_value: 3
+          - !ruby/object:Api::Type::Enum
+            name: sessionInitializationMode
+            description: |
+              The BFD session initialization mode for
+              this BGP peer. If set to `ACTIVE`, the Cloud Router will initiate the
+              BFD session for this BGP peer. If set to `PASSIVE`, the Cloud Router
+              will wait for the peer router to initiate the BFD session for this
+              BGP peer. If set to `DISABLED`, BFD is disabled for this BGP peer.
+            values:
+              - :ACTIVE
+              - :DISABLED
+              - :PASSIVE
+            required: true
+      - !ruby/object:Api::Type::String
+        name: 'enable'
+        description: |
+          The status of the BGP peer connection. If set to FALSE,
+          any active session with the peer is terminated and all associated
+          routing information is removed. If set to TRUE, the peer connection
+          can be established with routing information. The default is TRUE.
+        min_version: beta
+        default_value: "true"
```

Comment:
Does "true" work here or does it have to be all caps?

Reply:
Surprisingly it does. Regarding your errors this is because the feature is currently in closed Beta requiring project whitelisting, otherwise the fields are unavailable (and a reason why we shouldn't merge before the open Beta is launched, and the enable = TRUE by default will fail if the project is not whitelisted).